JavaScript/JavaScriptSDK.Tests/Contracts/Generated/RequestData.unittests.ts

```diff
@@ -12,6 +12,7 @@
 /// <reference path="../../../JavaScriptSDK/Contracts/Generated/PageViewData.ts" />
 /// <reference path="../../../JavaScriptSDK/Contracts/Generated/PageViewPerfData.ts" />
 /// <reference path="../../../JavaScriptSDK/Contracts/Generated/AjaxCallData.ts" />
+/// <reference path="../../../JavaScriptSDK/Contracts/Generated/AvailabilityData.ts" />
```

Comment:
This file was auto-generated by the BOND generator.
I'm not using this file, but I left all changes intact so everything will be insync with the same schema.